frontend/src/styles/globals.scss

```diff
@@ -53,6 +53,10 @@ body {
   -moz-osx-font-smoothing: grayscale;
 }
 
+button {
+  padding: 0;
+}
+
```

Comment:
suggestion (non-blocking): This seems like a pretty broad change. When this is merged on the dev server, can we ask QA to pay special attention to `button` elements across the site to make sure this change doesn't break any other buttons?

Reply:
Will make sure to let QA know.